.github/workflows/lockfileversion-check-v3.yml

```diff
@@ -0,0 +1,30 @@
+#check package-lock file version
+
+# With move to Node.js v18 and NPM 9, the resulting lockfileVersion is updated to 3.
+# This is a workflow to verify the use of NPM 9 and lockfileVersion 3
+# and intends to phase out the older workflow checking for lockfileVersion 2.
+
+name: lockfileVersion check v3
+
+on:
+  - workflow_call
+
+jobs:
+  version-check:
+    runs-on:  ubuntu-20.04
+    outputs:
```

Comment:
You shouldn't need to use the `outputs` option for current use case. You can simply create a GitHub env variable and use that in the next step. 
See following snippet for example (or [GitHub docs](url)):
```
    steps:
      - name: Set environment variable
        env:
          MY_VAR: "hello"
        run: echo "The variable value is $MY_VAR"
      
      - name: Modify environment variable
        run: echo "The new variable value is modified" > ${{ env.MY_VAR }}
      
      - name: Use environment variable
        run: echo "The variable value is now $MY_VAR"
```